ompi/mpi/fortran/mpif-h/ialltoallw_f.c

```diff
@@ -107,22 +108,32 @@ void ompi_ialltoallw_f(char *sendbuf, MPI_Fint *sendcounts,
     recvbuf = (char *) OMPI_F2C_BOTTOM(recvbuf);
 
     c_ierr = PMPI_Ialltoallw(sendbuf,
-                            OMPI_ARRAY_NAME_CONVERT(sendcounts),
-                            OMPI_ARRAY_NAME_CONVERT(sdispls),
-                            c_sendtypes,
-                            recvbuf,
-                            OMPI_ARRAY_NAME_CONVERT(recvcounts),
-                            OMPI_ARRAY_NAME_CONVERT(rdispls),
-                            c_recvtypes, c_comm, &c_request);
-    if (NULL != ierr) *ierr = OMPI_INT_2_FINT(c_ierr);
```

Comment:
Did you mean to remove this c_ierr/ierr check?
I see the check at the bottom for success, but I'd think we'd need to update a field in it if there is an error before we return???